src/Yoti.Auth/YotiProfile.cs

```diff
@@ -147,5 +153,36 @@ public YotiAttribute<DocumentDetails> DocumentDetails
                 return GetAttributeByName<DocumentDetails>(name: Constants.UserProfile.DocumentDetailsAttribute);
             }
         }
+
+        /// <summary>
+        /// Finds all the 'Age Over' and 'Age Under' derived attributes returned with the profile,
+        /// and returns them wrapped in <see cref="AgeVerification"/> objects.
+        /// Returns null if no matches were found.
+        /// </summary>
+        public ReadOnlyCollection<AgeVerification> AgeVerifications
+        {
+            get
+            {
+                return _ageVerificationParser.FindAllAgeVerifications();
+            }
+        }
+
+        /// <summary>
+        /// Searches for an <see cref="AgeVerification"/> corresponding to an `Age Under` check for the given age.
+        /// Returns null if no match was found.
+        /// </summary>
+        public AgeVerification AgeUnderVerification(int age)
```

Comment:
Again, maybe this should be _FindAgeUnderVerification_.

Maybe my brain is too Java-wired now, but this just doesn't look right.

Reply:
Same as above